library-backend/library-app/src/main/java/org/internship/library/app/security/CorsConfiguration.java

```diff
@@ -0,0 +1,61 @@
+package org.internship.library.app.security;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Configures CORS
+ */
+//@Configuration
+//@ConfigurationProperties(prefix = "cors")
+public class CorsConfiguration {
```

Comment:
Is this configuration used?

Reply:
It's not, It was an alternative to "CorsConfig" class when it gave me errors.